observability/src/main/java/io/grpc/observability/ObservabilityConfigImpl.java

```diff
@@ -113,12 +117,18 @@ public String getDestinationProjectId() {
   }
 
   @Override
-  public LogFilter[] getLogFilters() {
-    return logFilters;
+  public List<LogFilter> getLogFilters() {
+    if (logFilters == null) {
+      return null;
+    }
+    return ImmutableList.<LogFilter>builder().addAll(Arrays.asList(logFilters)).build();
```

Comment:
It would be better to store the filters and events as `ImmutableList` instead of just having the getters build from arrays.

Reply:
Updated to store filters and events as `ImmutableList`.